src/libsyntax/codemap.rs

```diff
@@ -799,7 +799,7 @@ impl CodeMap {
 
         let lo = self.lookup_char_pos_adj(sp.lo);
         let hi = self.lookup_char_pos_adj(sp.hi);
-        return (format!("{}:{}:{}: {}:{}",
+        return (format!("{}: {}:{} -> {}:{}",
```

Comment:
I think the first space shouldn't be there. Many editors let you say `open foo.rs:1:2` on the command line and will open focused on that line. It's easier to copy-paste if there is no space.